common/predictive-text/src/node/worker.ts

```diff
@@ -0,0 +1,13 @@
+import VirtualizedWorker from "./virtualizedWorker.js";
+import unwrap from '../unwrap.js';
+
+import { LMLayerWorkerCode } from "@keymanapp/lm-worker/build/lib/worker-main.wrapped-for-bundle.js";
+
+export default class DefaultWorker {
+  static constructInstance(): Worker {
+    let scriptStr = unwrap(LMLayerWorkerCode);
+    let worker = new VirtualizedWorker(scriptStr);
+
+    return worker as any as Worker;
```

Comment:
```suggestion
    return worker as any as Worker;
```

Yeah nah um I guess if you have to